src/main/java/mate/academy/service/UserService.java

```diff
@@ -2,6 +2,8 @@
 
 import mate.academy.model.User;
 
+import java.io.PushbackReader;
```

Comment:
why do we need this import?

Reply:
import java.io.PushbackReader was removed